cleanup.sh

```diff
@@ -167,7 +167,8 @@ while true; do
     if [[ "${next_token}" == "" ]]; then
         response=$( aws cognito-idp list-user-pools --max-results 1)
     else
-        response=$( aws cognito-idp list-user-pools --max-results 1 --starting-token "$next_token")
+        next_token=$( echo $next_token | sed 's/\"//g')
```

Comment:
If we're using `sed` here to get rid of the quotes (`"`), what do you think of using the `jq` argument [--raw-output / -r](https://stedolan.github.io/jq/manual/) when saving the `$next_token` value in line 197, so that the variable doesn't have the quotes?

Reply:
yes jq -r would work better - verified it locally. Updating ...